slide_analysis/splitting_service/splitting_service.py

```diff
@@ -1,33 +1,37 @@
-import openslide
-
-import slide_analysis.splitting_service.constants as constants
-from slide_analysis.utils.tile_class import Tile
+from slide_analysis.splitting_service.constants import *
+from slide_analysis.utils import Tile
+from slide_analysis.utils.tile_stream_class import TileStream
 
 
 class SplittingService:
     def __init__(self):
-        self.tile_width = constants.BASE_TILE_WIDTH
-        self.tile_height = constants.BASE_TILE_HEIGHT
-
-    def _open_image(self, filename):
-        slide = openslide.open_slide(filename)
```

Comment:
why remove opening with openslide? This service is the first one addressed by user. Do you expect user to send you alread opened slide?